### ISSUE TITLE
vignette not building for me

### ISSUE THREAD

**Issue by wcornwell** [mwpennell/phyndr-ms#31]
After calling `remake::make`:

```R
[    OK ] source_data/zanne.tre
[  KNIT ] vignette/vignette.md            |  knitr::knit("vignette/vignette.Rmd", "vignette/vignette.md")
Quitting from lines 143-145 (vignette.Rmd) 
Restoring previous version of vignette/vignette.md
Error in phyndr_taxonomy_cleanup(phy2, data_species2) : 
  Zero overlap in data_species and phy
In addition: Warning message:
In file.create(to[okay]) :
  cannot create file 'vignette/vignette.md', reason 'No such file or directory'
```

**Comment by wcornwell** [comment]
seems to be something about the row names in `"../source_data/bmr.csv"` but not sure how you're generating that file.

**Comment by mwpennell** [comment]
This is a bit strange. I just pulled a clean version of the repo in another directory and it ran seamlessly. I made a minor edit in https://github.com/mwpennell/phyndr-ms/commit/e345b552f82c837edcb5e8d002157e6fce501fb7 but i don't know why this would affect things. can you try cloning the repo again and running it fresh.

**Comment by wcornwell** [comment]
I think it might be broken with a newer version of `readxl`.  with the tbl_df objects the row names get dropped when you select columns in this way...

```
> filename <- "source_data/bmr_data_test.xls"
>   b <- read_excel(filename, skip=6)
>   rownames(b) <- sapply(b[,"Genus Species"], function(x) gsub(" ", "_", x))
>   head(rownames(b))
[1] "Tachyglossus_aculeatus"   "Zaglossus_bartoni"       
[3] "Ornithorhynchus_anatinus" "Caluromys_derbianus"     
[5] "Gracilinanus_microtarsus" "Thylamys_elegans"        
>   b <- b[,c("Mass (g)", "BMR (W)")]
>   head(rownames(b))
[1] "1" "2" "3" "4" "5" "6"
```

**Comment by mwpennell** [comment]
Ok, I'll check this.

*shakes fist at the air*

**Comment by wcornwell** [comment]
I think I fixed it here: 430e30bf67441f73a9bc91bb2a1f9d541b266652 

check out to make sure it works for you

**Comment by mwpennell** [comment]
perfect. works fine. thanks @wcornwell !
